### DISCUSSION TITLE
Formula as display value [Mar 27]

### DISCUSSION THREAD

**Discussion by dstala** [nocodb/nocodb#5406] [Discord Threads]
> 🙋‍♂️ **ibarot**

**Comment by o1lab** [comment]
> 👨‍💻 **pranavxc**  
#### Can you share the formula that you had created ? Does the formula works if you are setting it as DisplayValue ?  
#  
> 🙋‍♂️ **ibarot**  
#### In Table MaterialSize : Formula is a CONCAT({MaterialType}, " ", {Size}).  Formula works when I set it as a DisplayValue. I can see the concatenated values in MaterialMaster table which has many to one relationship with MaterialSize.  
#  
> 👨‍💻 **pranavxc**  
#### Is `MaterialType` column used in formula is a Lookup ?  
#  
> 🙋‍♂️ **ibarot**  
#### No, Not a lookup. MaterialSize has a relationship many to one with MaterialType  
#  
> 🙋‍♂️ **ibarot**  
#### The value in Column MaterialType comes from table MaterialType, but I did not use it as a lookup field  
#  
> 👨‍💻 **pranavxc**  
#### ok thanks for the quick reply, let me try to reproduce the issue locally  
#  
> 🙋‍♂️ **ibarot**  
#### sure, thanks for your help. If you need any more details, do let me know  
#  
> 👨‍💻 **pranavxc**  
#### Can you share the erd of your db for better understanding or the model meta data also fine  
#  
> 👨‍💻 **pranavxc**  
#### Is there a column named `MaterialTypeSize` in `MaterialSize` table ?  
#  
> 🙋‍♂️ **ibarot**  
####   
![image0](https://cdn.discordapp.com/attachments/1078606759238836304/1078981501150048346/image.png)  
#  
> 🙋‍♂️ **ibarot**  
#### MaterialTypeSize is the formula column that I created  
#  
> 🙋‍♂️ **ibarot**  
#### It is set as DisplayValue in MaterialSize table  
#  
> 👨‍💻 **pranavxc**  
#### ok thanks for sharing, let me try to replicate it  
#  
> 👨‍💻 **pranavxc**  
#### Looks like it's an issue with formula and not related to display value, can you create an issue in our Github with above details.  
#  
> 🙋‍♂️ **ibarot**  
#### ok...let me try doing that..  
#  
> 🙋‍♂️ **ibarot**  
#### Thanks a lot.... I will post all the details I shared here.  
#  

 [Autogenerated from discord](https://discord.com/channels/661905455894888490/1078606759238836304)  
- - - -  
#### Join [NocoDB's](http://www.nocodb.com/)  community  
[Discord](https://discord.gg/5RgZmkW) • [Twitter](https://twitter.com/nocodb) • [Reddit](https://www.reddit.com/r/NocoDB/) • [Docs](https://docs.nocodb.com/)
